### PR TITLE
Add GetInt64

### PR DESCRIPTION
**Pull request by ChillerDragon**
Comes in handy for my downstream

### CODE REVIEW

File: src/engine/server/databases/mysql.cpp [hunk 1]
```diff
@@ -550,6 +551,34 @@ int CMysqlConnection::GetInt(int Col)
 	return Value;
 }
 
+int64_t CMysqlConnection::GetInt64(int Col)
+{
+	Col -= 1;
+
+	MYSQL_BIND Bind;
+	int Value;
```
Comment (def-) [inline src/engine/server/databases/mysql.cpp:559]: int64_t

**Review by def- — CHANGES_REQUESTED** [review]

**Review by def- — COMMENTED** [review]
lgtm. Did you test if this actually works with large numbers?

**Comment by ChillerDragon** [comment]
nope untested

Am So., 7. Nov. 2021 um 17:06 Uhr schrieb Dennis Felsing <
***@***.***>:

**Comment by heinrich5991** [comment]
Please test this.

**Comment by ChillerDragon** [comment]
Okay i tested with sqlite3 and successfully loaded the value ``222147483647`` into a ``int64_t``

``std::numeric_limits<int>::max()`` gave me ``2147483647``

**Pull request merged**